### DISCUSSION TITLE
Personal access token with a repository scope

### DISCUSSION THREAD

**Discussion by peaceiris** [community/community#5081] [Other Feature Feedback, Questions, & Ideas] [answered]
Today, a personal access token will be generated with permissions for all user repositories (e.g. `public_repo` scope for all public repositories).

We need a more narrow scope for a PAT to decrease the security risk.

Related: https://github.com/peaceiris/actions-label-commenter/pull/485

**Comment by Katsute** [comment, marked as answer]
https://github.com/github/roadmap/issues/184

> - Scoping access to specific repos and orgs

**Reply by peaceiris** [reply]
Nice. Thank you!

**Comment by Chekote** [comment]
Were you ever able to get this to work? I am trying to do the same thing, but the fine-grained tokens only let me choose "all public" or "all":
![image](https://github.com/community/community/assets/550486/edfae1a2-51f3-4672-b902-53678e83f4a4)
